pom.xml

```diff
@@ -20,7 +20,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <mysqlConnectorVersion>8.0.16</mysqlConnectorVersion>
+        <mysqlConnectorVersion>8.0.28</mysqlConnectorVersion>
```

Comment:
Did you verify that tests still pass with no regression? It's not good to apply this dependency upgrades without at least running tests

Reply:
No, but I thought about the implications of merging this 😉 
Here was my thinking:
1. This is adopting a patch release of of mysql-connector-java v8.0 (8.0.16 → 8.0.28), so there is a high likelihood of being ok.
2. We _just_ released openmrs-eip 3.1.0, so this is one of the many things that will have plenty of time to be tested while working on 3.2.0-SNAPSHOT.

Reply:
Ok